scene/animation/animation_player.cpp

```diff
@@ -572,7 +572,7 @@ void AnimationPlayer::_animation_process_data(PlaybackData &cd,float p_delta,flo
 
 	} else {
 		
-		if (next_pos<0 or next_pos>len) {
+        if (next_pos<0 || next_pos>len) {
```

Comment:
Thanks for the hints.

I'm using QtCreator, so it's Tools -> Options -> Text Editor -> Behavior and under "Tab Policy", change to "Tabs Only". Am I right?